### DISCUSSION TITLE
hostname/domain when running in home network

### DISCUSSION THREAD

**Discussion by HubKing** [emersion/maddy#551] [Q&A] [answered]
If it were a public server, the domain would be something like "example.org", and the host name would be something like "mail.example.org", in which the first part indicates the server's name, the second part indicates the organisation or company, and the third one is just type of the company. It makes sense. 

But what if I am using Maddy inside of my home network to exchange e-mails only amongst the users of the Maddy server. That is, mails never go to/come another server)? Let's say that the name of the Linux PC on which Maddy is running is "XXX", then, \$ (hostname) = XXX, and \$ (primary_domain) is XXX, too?

Ideally, the e-mail addresses I would like to use are something like "user1@XXX", "user2@XXX".

**Comment by foxcpp** [comment, marked as answer]
>But what if I am using Maddy inside of my home network to exchange e-mails only amongst the users of the Maddy server. That is, mails never go to/come another server)?

In this case you are free to set hostname and domain to literally any value. Probably it would be a good idea to avoid collisions with public domain names  though. So something like hubking.local would work prefectly.
